bindgen-integration/cpp/Test.cc

```diff
@@ -83,22 +118,80 @@ Fifth::assert(unsigned short nWeekDay,
               unsigned short nYear,
               unsigned char byte)
 {
-  return this->nWeekDay == nWeekDay &&
-      this->nMonthDay == nMonthDay &&
-      this->nMonth == nMonth &&
-      this->nYear == nYear &&
-      this->byte == byte;
+    printf("\n");
+    printf("Fifth::assert(%d, %d, %d, %d, %d)\n",
+           nWeekDay,
+           nMonthDay,
+           nMonth,
+           nYear,
+           byte);
+    printf("       actual(%d, %d, %d, %d, %d)\n",
+           this->nWeekDay,
+           this->nMonthDay,
+           this->nMonth,
+           this->nYear,
+           this->byte);
+    return this->nWeekDay == nWeekDay &&
+        this->nMonthDay == nMonthDay &&
+        this->nMonth == nMonth &&
+        this->nYear == nYear &&
+        this->byte == byte;
 }
 
 bool
 Sixth::assert(unsigned char byte,
               unsigned char nWeekDay,
               unsigned char nMonth,
               unsigned char nMonthDay) {
-  return this->nWeekDay == nWeekDay &&
-      this->nMonthDay == nMonthDay &&
-      this->nMonth == nMonth &&
-      this->byte == byte;
+    printf("\n");
```

Comment:
Not sure how worth are the printfs? i'd usually just use `gdb` for this.

Reply:
@emilio Agreed. Better to delete this clutter. It's trivial to add back in for debugging if someone doesn't have `gdb`.